src/main/java/in/ac/bits/protocolanalyzer/persistence/repository/SaveRepository.java

```diff
@@ -144,4 +150,20 @@ private void publishEndOfSave(long time) {
 		//log.info("Publishing end of Save Repository");
 		eventBus.post(new SaveRepoEndEvent(time));
 	}
-}
+
```

Comment:
Since lowWaterMark is a field, lombok provides getters and setters on the same. We need not create these methods.